Src/Test.UnitTests.Sarif.PatternMatcher.Sdk/Test.UnitTests.Sarif.PatternMatcher.Sdk.csproj

```diff
@@ -14,6 +14,7 @@
   
   <ItemGroup>
     <ProjectReference Include="..\Sarif.PatternMatcher.Sdk\Sarif.PatternMatcher.Sdk.csproj" />
+    <ProjectReference Include="..\Sarif.PatternMatcher\Sarif.PatternMatcher.csproj" />
```

Comment:
>PatternMatcher [](http://example.com/codeflow?start=61&length=14)

instead of adding this dependency, add the tests that require this to sarif.patternmatches tests

Reply:
My purpose is to test ValidationResult related logic, which is in Sarif.PartternMatcher.Sdk. In the test i have to call SearchSkimmer/AnalyzeContext which are in Sarif.PatternMatcher. Where should I place these tests?

Reply:
move to the sarif.pattern.matcher.
why: u are actually validating the logic in sarifpatternmatcher and not the class itself.